UltiSnips/generate.py

```diff
@@ -353,8 +364,16 @@ def get_collection_name(filepath:str) -> str:
             docstring_builtin['collection_name'] = "ansible.builtin"
             modules_docstrings.append(docstring_builtin)
 
+    system_modules_paths = get_files_collections()
+    for f in system_modules_paths:
+        docstring_system = get_module_docstring(f)
+        if docstring_system and docstring_system not in modules_docstrings:
+            collection_name = get_collection_name(f)
+            docstring_system['collection_name'] = collection_name
+            modules_docstrings.append(docstring_system)
+
     if args.user:
```

Comment:
The default behavior for `--user` remains the same since `get_files_collections(user=True)` is behind this if statement.